posthog/queries/query_date_range.py

```diff
@@ -174,21 +174,40 @@ def date_from(self) -> Tuple[str, Dict]:
 
         return date_from_query, date_from_param
 
-    def _get_timezone_aware_date_condition(self, operator: str, date_clause: str) -> str:
-        if self.should_round:
-            return f"AND toDateTime({self._table}timestamp, 'UTC') {operator} {self._timezone_date_clause(date_clause)}"
+    def _get_timezone_aware_date_condition(self, operator: Literal[">=", "<="]) -> str:
+        relevant_param = "date_from" if operator.startswith(">") else "date_to"
```

Comment:
nit: a bit confusing maybe to determine the field from the operator argument, but it's fine

Reply:
Agreed, inverted this to make `date_param: Literal["date_from", "date_to"]` the argument instead.